tests/compile-fail/wildcard_before_input_state.rs

```diff
@@ -0,0 +1,12 @@
+extern crate smlang;
+
+use smlang::statemachine;
+
+statemachine! {
+    transitions: {
+        _ + Event1 = Fault, //~ State and event combination specified multiple times, remove duplicates.
```

Comment:
This is a compile-fail check. Can we also add a tests including the now-accepted usage?

Reply:
I added a test in `test.rs`. Do you mean you prefer something under `examples`?

Reply:
Ah, I wrote this immediately before seeing that. That's fine with me :)